Doc/reference/datamodel.rst

```diff
@@ -2195,8 +2195,10 @@ through the object's keys; for sequences, it should iterate through the values.
    :const:`NotImplemented`, which is treated the same as if the
    ``__length_hint__`` method didn't exist at all. This method is purely an
    optimization and is never required for correctness.
+   Used by functions like :func:`list` to choose an initial allocation size.
 
    .. versionadded:: 3.4
+      Originally specified by :pep:`424`.
 
```

Comment:
In other places the wording is "Originally specified in PEP XXX"  (in and not by).